### ISSUE TITLE
Upscayl error processing on file paths with non-English characters

### ISSUE THREAD

**Issue by Nancy064** [upscayl/upscayl#628]
### Checklist

- [X] I have checked that this issue isn't addressed in [the FAQ](https://github.com/upscayl/upscayl#-faq).
- [X] I have checked that this issue isn't addressed in any other issue or pull request.
- [X] I have tried [setting my GPU ID](https://github.com/upscayl/upscayl/wiki/Guide#gpu-id).
- [ ] I am using Windows and I've tried [common troubleshooting steps](https://github.com/upscayl/upscayl/wiki/Troubleshooting).

### Describe the Bug

Upscayl error processing (scaling and converting) the image. Upscayl load to more than 90% then error message.

### To Reproduce

Try to upscale anything with a model on Windows 10 Pro 22H2

### Version (or commit hash)

v2.9.5

### Select your Operating System

Windows

### OS Version

Windows 10 Pro 22H2

### GPU Name

NVIDIA Geforce RTX 3050

### Paste the logs

```sh
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Getting gpuId from localStorage:  0
🔀 Setting model to 
🐞 BACKEND REPORTED:  💾 Updating Save Output Folder:  false
🔀 Setting model to realesrgan-x4plus
🐞 BACKEND REPORTED:  📝 Updating Overwrite:  false
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Getting gpuId from localStorage:  0
⚙️ Getting rememberOutputFolder from localStorage:  false
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Getting gpuId from localStorage:  0
🔀 Setting model to 
🔀 Setting model to realesrgan-x4plus
🔄 Resetting image paths
⤵️ Dropped file:  {"type":"image/jpeg","filePath":"H:\\1. DIỆP\\Best88\\stock\\girl\\24862510_157838044971742_5307184528451455562_n.jpg","extension":"jpg"}
🖼 Setting image path:  H:\1. DIỆP\Best88\stock\girl\24862510_157838044971742_5307184528451455562_n.jpg
🗂 Setting output path:  H:\1. DIỆP\Best88\stock\girl
🖼 imagePath:  H:\1. DIỆP\Best88\stock\girl\24862510_157838044971742_5307184528451455562_n.jpg
🔤 Extension:  jpg
🔄 Resetting Upscaled Image Path
🏁 UPSCAYL
🐞 BACKEND REPORTED:  📝 Updating Overwrite:  false
🐞 BACKEND REPORTED:  🖼️ Updating No Image Processing:  false
🐞 BACKEND REPORTED:  📐 Updating Compression:  0
🐞 BACKEND REPORTED:  ✅ Upscayl Variables:  {"model":"realesrgan-x4plus","gpuId":"0","saveImageAs":"png","inputDir":"H:\\1. DIỆP\\Best88\\stock\\girl","outputDir":"H:\\1. DIỆP\\Best88\\stock\\girl","fullfileName":"24862510_157838044971742_5307184528451455562_n.jpg","fileName":"24862510_157838044971742_5307184528451455562_n","initialScale":"4","desiredScale":"2","outFile":"H:\\1. DIỆP\\Best88\\stock\\girl\\24862510_157838044971742_5307184528451455562_n_upscayl_2x_realesrgan-x4plus.png","compression":0}
🐞 BACKEND REPORTED:  📢 Upscayl Command:  -i,H:\1. DIỆP\Best88\stock\girl\24862510_157838044971742_5307184528451455562_n.jpg,-o,H:\1. DIỆP\Best88\stock\girl\24862510_157838044971742_5307184528451455562_n_upscayl_2x_realesrgan-x4plus.png,-s,4,-m,C:\Program Files\Upscayl\resources\models,-n,realesrgan-x4plus,-g,0,-f,png
🐞 BACKEND REPORTED:  👶 Updating Child Processes:  {"binary":"C:\\Program Files\\Upscayl\\resources\\bin\\upscayl-bin","args":["C:\\Program Files\\Upscayl\\resources\\bin\\upscayl-bin","-i","H:\\1. DIỆP\\Best88\\stock\\girl\\24862510_157838044971742_5307184528451455562_n.jpg","-o","H:\\1. DIỆP\\Best88\\stock\\girl\\24862510_157838044971742_5307184528451455562_n_upscayl_2x_realesrgan-x4plus.png","-s","4","-m","C:\\Program Files\\Upscayl\\resources\\models","-n","realesrgan-x4plus","-g","0","-f","png"]}
🐞 BACKEND REPORTED:  🛑 Updating Stopped:  false
🐞 BACKEND REPORTED:  image upscayl:  [0 NVIDIA GeForce RTX 3050]  queueC=2[8]  queueG=0[16]  queueT=1[2]
[0 NVIDIA GeForce RTX 3050]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 NVIDIA GeForce RTX 3050]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[0 NVIDIA GeForce RTX 3050]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🚧 UPSCAYL_PROGRESS:  [0 NVIDIA GeForce RTX 3050]  queueC=2[8]  queueG=0[16]  queueT=1[2]
[0 NVIDIA GeForce RTX 3050]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 NVIDIA GeForce RTX 3050]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[0 NVIDIA GeForce RTX 3050]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🐞 BACKEND REPORTED:  image upscayl:  0.00%

🚧 UPSCAYL_PROGRESS:  0.00%

🐞 BACKEND REPORTED:  image upscayl:  4.00%

🚧 UPSCAYL_PROGRESS:  4.00%

🐞 BACKEND REPORTED:  image upscayl:  8.00%

🚧 UPSCAYL_PROGRESS:  8.00%

🐞 BACKEND REPORTED:  image upscayl:  12.00%

🚧 UPSCAYL_PROGRESS:  12.00%

🐞 BACKEND REPORTED:  image upscayl:  16.00%

🚧 UPSCAYL_PROGRESS:  16.00%

🐞 BACKEND REPORTED:  image upscayl:  20.00%

🚧 UPSCAYL_PROGRESS:  20.00%

🐞 BACKEND REPORTED:  image upscayl:  24.00%

🚧 UPSCAYL_PROGRESS:  24.00%

🐞 BACKEND REPORTED:  image upscayl:  28.00%

🚧 UPSCAYL_PROGRESS:  28.00%

🐞 BACKEND REPORTED:  image upscayl:  32.00%

🚧 UPSCAYL_PROGRESS:  32.00%

🐞 BACKEND REPORTED:  image upscayl:  36.00%

🚧 UPSCAYL_PROGRESS:  36.00%

🐞 BACKEND REPORTED:  image upscayl:  40.00%

🚧 UPSCAYL_PROGRESS:  40.00%

🐞 BACKEND REPORTED:  image upscayl:  44.00%

🚧 UPSCAYL_PROGRESS:  44.00%

🐞 BACKEND REPORTED:  image upscayl:  48.00%

🚧 UPSCAYL_PROGRESS:  48.00%

🐞 BACKEND REPORTED:  image upscayl:  52.00%

🚧 UPSCAYL_PROGRESS:  52.00%

🐞 BACKEND REPORTED:  image upscayl:  56.00%

🚧 UPSCAYL_PROGRESS:  56.00%

🐞 BACKEND REPORTED:  image upscayl:  60.00%

🚧 UPSCAYL_PROGRESS:  60.00%

🐞 BACKEND REPORTED:  image upscayl:  64.00%

🚧 UPSCAYL_PROGRESS:  64.00%

🐞 BACKEND REPORTED:  image upscayl:  68.00%

🚧 UPSCAYL_PROGRESS:  68.00%

🐞 BACKEND REPORTED:  image upscayl:  72.00%

🚧 UPSCAYL_PROGRESS:  72.00%

🐞 BACKEND REPORTED:  image upscayl:  76.00%

🚧 UPSCAYL_PROGRESS:  76.00%

🐞 BACKEND REPORTED:  image upscayl:  80.00%

🚧 UPSCAYL_PROGRESS:  80.00%

🐞 BACKEND REPORTED:  image upscayl:  84.00%

🚧 UPSCAYL_PROGRESS:  84.00%

🐞 BACKEND REPORTED:  image upscayl:  88.00%

🚧 UPSCAYL_PROGRESS:  88.00%

🐞 BACKEND REPORTED:  image upscayl:  92.00%

🚧 UPSCAYL_PROGRESS:  92.00%

🐞 BACKEND REPORTED:  image upscayl:  96.00%

🚧 UPSCAYL_PROGRESS:  96.00%

🐞 BACKEND REPORTED:  💯 Done upscaling
🐞 BACKEND REPORTED:  ♻ Scaling and converting now...
🐞 BACKEND REPORTED:  📐 Processing Image:  {"originalWidth":911,"originalHeight":885,"scale":"2","saveImageAs":"png","compressionPercentage":0,"compressionLevel":0}
🐞 BACKEND REPORTED:  ❌ Error processing (scaling and converting) the image. Please report this error on GitHub. Error: Input file is missing: H:\1. DIỆP\Best88\stock\girl\24862510_157838044971742_5307184528451455562_n_upscayl_2x_realesrgan-x4plus.png
🔄 Resetting image paths
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Getting gpuId from localStorage:  0
⚙️ Getting rememberOutputFolder from localStorage:  false
```

**Comment by NayamAmarshe** [comment]
Please use a better name for the folder.

**Comment by DinandDotNet** [comment]
Same problem. used batch mode and got an error from a url in the map. 
The error happends, but can Upscayl add functionality to still process the png to jpg. That would help a lot.

**Comment by NayamAmarshe** [comment]
Unfortunately, the fix for now is to only place the files in a folder with English alphabets in its name.
So you can do it in C:\ drive, create a folder named 'test' there and give it a try.

**Comment by Nancy064** [comment]
> Unfortunately, the fix for now is to only place the files in a folder with English alphabets in its name. So you can do it in C:\ drive, create a folder named 'test' there and give it a try.

Thank you so much!
It's working when I change folder name with English alpabets.

**Comment by aaronliu0130** [comment]
Okay, regression yet again (#452 , #217)

**Comment by NayamAmarshe** [comment]
This will need to be fixed in upscayl-ncnn repository. I'm assigning this to @TGS963 at the moment.

**Comment by TGS963** [comment]
> This will need to be fixed in upscayl-ncnn repository. I'm assigning this to @TGS963 at the moment.

Right, this can be fixed. Thanks for the flag

**Comment by NayamAmarshe** [comment]
Fixed in v2.11
